frontend/src/metabase/nav/components/SearchResults.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+import Search from "metabase/entities/search";
+import Icon from "metabase/components/Icon";
+import SearchResult from "metabase/search/components/SearchResult";
+
+const SEARCH_LIMIT = 50;
+
+const propTypes = {
+  searchText: PropTypes.string,
+};
+
+export const SearchResults = ({ searchText }) => {
+  return (
+    <Search.ListLoader
+      query={{ q: searchText, limit: SEARCH_LIMIT }}
+      wrapped
+      reload
+      debounced
+    >
+      {({ list }) => {
+        const hasResults = list.length > 0;
+
+        return (
+          <ol data-testid="search-results-list">
+            {hasResults ? (
+              list.map(item => (
+                <li key={`${item.model}:${item.id}`}>
+                  <SearchResult result={item} compact={true} />
+                </li>
+              ))
+            ) : (
+              <li className="flex flex-column align-center justify-center p4 text-medium text-centered">
```

Comment:
Maybe consider converting from className styles to a styled component.

Reply:
Used EmptyState here